app/grants/views.py

```diff
@@ -518,7 +517,7 @@ def grant_fund(request, grant_id, grant_slug):
             'grant': grant,
             'text': _('You already have an active subscription for this grant.')
         }
-        return TemplateResponse(request, 'grants/shared/error.html', params)
+        # return TemplateResponse(request, 'grants/shared/error.html', params)
```

Comment:
Why was this commented out ?

Reply:
:scream: Sorry, I commented this to reuse some grants, but I comment it again!